frontend/app/src/components/EthNamesHint.vue

```diff
@@ -6,11 +6,13 @@ const visible = ref(false);
 
 <template>
   <div>
-    <RuiButton id="eth-names-hint" variant="text" icon @click="visible = true">
-      <RuiIcon name="question-line" />
-    </RuiButton>
-    <VMenu v-model="visible" offset-x nudge-right="25" attach="#eth-names-hint">
-      <div class="pa-4 min-w-[20rem] flex flex-col gap-2">
+    <VMenu v-model="visible" offset-x>
+      <template #activator="{ on }">
+        <RuiButton variant="text" icon @click="visible = true" v-on="on">
+          <RuiIcon name="question-line" />
+        </RuiButton>
+      </template>
```

Comment:
I really have no idea why it did not work the first time.

I was not getting the button at all.